.circleci/config.yml

```diff
@@ -0,0 +1,87 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_EMAIL   - login info for docker hub
+# DOCKER_USER
+# DOCKER_PASS
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: docker:17.12.1-ce
+    working_directory: /dockerflow
+    steps:
+      - checkout
```

Comment:
you'll want to include this to avoid problems on circleci: https://github.com/mozilla-services/megaphone/commit/f81d556a415d7e775e358373df22e125d0c37406